### ISSUE TITLE
tinymce with codesample & Text.pre2htmlentities

### ISSUE THREAD

**Issue by vait** [bludit/bludit#406]
I try to use tinymce plugin with codesample, but I am having a issue on rendering post content: it render without highlight. I found that in class Text in function pre2htmlentities section with &lt;pre&gt;-tag's attributes is missing.
I think it must be like that:

>  return preg_replace_callback('/<pre.*?><code(.*?)>(.*?)<\/code><\/pre>/imsu',
>         create_function('$input', 'return "<pre **$input[0]**><code $input[1]>".htmlentities($input[2])."</code></pre>";'),
>          $string);

**Comment by vait** [comment]
I made mistake. There is right code:

> return preg_replace_callback('/<pre **(.*?)** ><code(.*?)>(.*?)<\/code><\/pre>/imsu',
>         create_function('$input', 'return "<pre **$input[1]**><code **$input[2]**>".htmlentities(**$input[3]**)."</code></pre>";'),
>         $string);

**Comment by clickwork-git** [comment]
Because the problem is related to TinyMCE you can open an issue at

https://github.com/tinymce/tinymce

**Comment by vait** [comment]
I don't think so because in post-file data saved correctly with necessary attributes.

```
<pre class="language-php"><code>&lt;?php
    // Plugins Site Head
    Theme::title();
    Theme::plugins('siteHead');
?&gt;</code></pre>
```
But on reading and rendering the attributes are lost.

**Comment by clickwork-git** [comment]
You cannot use code with TinyMCE. It's converted to HTML entities.

**Comment by vait** [comment]
No no no. I try to explain to you:
TinyMCE + codesample plugin generates code section with highlight. 
![screen1](https://cloud.githubusercontent.com/assets/5323700/24977298/40d35f72-1fd5-11e7-8c99-5090292b8291.PNG)

It save html entities without highlight because prism.js makes highlight on client-side and not modifies source of post. 
![screen2](https://cloud.githubusercontent.com/assets/5323700/24977297/40d3289a-1fd5-11e7-953f-ce5693067dc5.PNG)

But on rendering post Text.pre2htmlentities function is loosing ONLY pre-tag attributes, and all remaining code is rendering correctly.
![screen3](https://cloud.githubusercontent.com/assets/5323700/24977299/40d87fac-1fd5-11e7-9c08-4d1441177bc0.PNG)

**Comment by dignajar** [comment]
There was some problem with ParsedownExtra, I fix it on the version 2.0.
